Doc/library/unittest.rst

```diff
@@ -2012,13 +2012,16 @@ Loading and running tests
 
    .. method:: wasSuccessful()
 
-      Return ``True`` if all tests run so far have passed, otherwise returns
-      ``False``.
+      Return ``True`` if all tests run so far have passed and at least one test
+      was found, otherwise returns ``False``.
```

Comment:
The code checks testsRun + testsSkipped > 0, so it's not exactly "was found" (what if many were found but we haven't started running anything yet?)

nit: I think it's easier to understand if the sentence lists the conditions chronologically, something like: "at least one test has been executed or skipped and no test has failed so far".